README.md

```diff
@@ -7,53 +6,90 @@ At the LAO, we realized that even though Moloch is very useful and powerful, it
 
 This is why we would like to introduce a more modular approach to Moloch architecture, which will give us:
 
-- Simpler code, each part would do something more specific, this means easier to understand
-- Adaptable, we will be able to adapt each part of the DAO to the needs of the ones using it without the need to audit the entire code bade every time
-- Upgradable, it should be easier to upgrade parts once the need evolves. The best example we have in mind is voting, maybe the way of voting evolve with time and it is good to be able to upgrade that economic, we can imagine some modules being used by multiple DAOs without the need to be redeployed
+- Simpler code - each part would do something more specific, this means easier to understand.
+- Adaptability - we will be able to adapt each part of the DAO to the needs of the ones using it without the need to audit the entire code base every time.
+- Upgradability - it should be easier to upgrade parts once the need evolves. The best example we have in mind is voting. Maybe the way of voting evolves with time and it is good to be able to upgrade that economic. We can imagine some modules being used by multiple DAOs without the need to be redeployed.
 
-Inspired by the hexagonal architecture pattern we believe that we can have additional layers of security and break the main contract into smaller contracts. With that, we create loosely coupled modules/contracts, easier to audit, and can be easily connected to the DAO.
+Inspired by the hexagonal architecture pattern we believe that we can have additional layers of security, and break the main contract into smaller contracts. With that, we create loosely coupled modules/contracts, easier to audit, and can be easily connected to the DAO.
```

Comment:
We should add a reference to background information about the hexagonal design pattern.  It's an oddly named design pattern (since it is not dependent on the hexagon shape at all) and may be confusing to people who haven't been exposed to it before.

https://en.wikipedia.org/wiki/Hexagonal_architecture_(software)

Reply:
Good call. Just updated. Thanks